utils/src/ast-grep/resolve-binding-path.ts

```diff
@@ -68,6 +68,31 @@ function resolveBindingPathRequire(node: SgNode<Js>, path: string) {
 		});
 	}
 
+	const propertyAccesses = activeNode.findAll({
+		rule: {
+			kind: "property_identifier",
+			inside: {
+				kind: "member_expression",
+			},
+		},
+	});
+
+	if (propertyAccesses.length) {
+		const pathArr = path.split(".");
+		let newPath = ["$"];
```

Comment:
nit: I think `newPath` is actually constant?

Reply:
Good, let me update it